guides/overview-users.md

```diff
@@ -1,8 +1,8 @@
-# Users & Contacts
+# Members & Contacts
 
-Work with your site’s contacts and registered users. These APIs allow you to identify the current user, log users in and out, create new contacts, send triggered emails, and more.
+Work with your site’s contacts and members. These APIs allow you to identify the current member, log members in and out, create new contacts, send triggered emails, and more.
 
-*   **[wix-users](https://www.wix.com/velo/reference/wix-users.html)**
-*   **[wix-users-backend](https://www.wix.com/velo/reference/wix-users-backend.html)**
+*   **[wix-members](https://www.wix.com/velo/reference/wix-members-backend)**
```

Comment:
The link is wrong - it is going to wix-members-backend instead of wix-members.

Reply:
Thanks!!!! The new commit fixes that 😄